website/source/docs/platform/k8s/helm/examples/standalone-audit.html.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: "docs"
+page_title: "Standalone Server with Audit Storage"
+sidebar_current: "docs-platform-k8s-examples-standalone-audit"
+sidebar_title: "Standalone Server with Audit Storage"
+description: |-
+  Describes how to set up a standalone Vault with audit storage
+---
+
+# Standalone Server with Audit Storage
+
+~> **Important Note:** This chart is not compatible with Helm 3. Please use Helm 2 with this chart.
+
+The below `values.yaml` can be used to set up a single server Vault cluster with
+auditing enabled.
+
+```yaml
+global:
+  enabled: true
+  image: "vault:1.3.0"
```

Comment:
Old version

Reply:
Also, this doesn't exist anymore. It's now under server. I suggest just removing the global stuff in general.

Reply:
Removed